src/register-org/models/registrationdata.model.ts

```diff
@@ -10,12 +11,13 @@ export type RegistrationData = {
   hasPBA: boolean;
   contactDetails: ContactDetails;
   companyHouseNumber?: string;
-  address: string;
+  address: AddressModel;
   organisationType: string;
   otherOrganisationType?: string;
   otherOrganisationDetail?: string;
   organisationNumber?: string;
   regulatorRegisteredWith: string;
+  inInternationalMode: boolean;
```

Comment:
Typo? should be "isInternationalMode"?

Reply:
Not really, inInternationalMode makes more sense as the address component can have the international mode but not be in international mode. In international mode makes more sense on session storage (relevant to this data) as it is simply to confirm whether the component is already in international mode.